Oddities.WinHelp/Topics/TopicFile.cs

```diff
@@ -1,9 +1,9 @@
-﻿using JetBrains.Annotations;
+using JetBrains.Annotations;
 
 namespace Oddities.WinHelp.Topics;
 
 [PublicAPI]
-public struct TopicFile
+public readonly record struct TopicFile
```

Comment:
Since this particular struct has private fields, it doesn't make much sense in converting it to a `record struct`. Let's leave it as it was before.

Reply:
Done